lib/membrane/video_compositor/scene/texture.ex

```diff
@@ -0,0 +1,24 @@
+defmodule Membrane.VideoCompositor.Scene.Texture do
```

Comment:
What is the genesis of a `Texture` name? :D 
To be honest, it's not intuitive for me to call it that way - as far as I know, a Texture is just a sequence of Transformations (with an ability to set a "fixed" output resolution).

Reply:
I guess as some sort of mirror to `Canvas` object: both have set of transformations and use single input, but one operates on canvases, the other - on textures. 

Name might be a little redundant for now, but I guess it describes what's under the hood. You've got your texture on GPU from `:input`, you transform it according to the `:transformations` and you want the output texture to have resolution specified under the `:resolution`.

Reply:
As @DominikWolek explained, the name describes what is happening under the hood. In my first design, I used `Transformations` and `Layouts` as processing graph nodes, which took `Textures` as inputs. Since many times we chain multiple single-input single-output augmentations  (like cropping and corner rounding), it's more compact to have a node that can perform any number of such operations, than having to define each of them separately.

It's also why `Transformations` are not `Objects`.  We wrap them into `Textures` in `Scene` `objects`.

I think that the `Texture` name clarifies why we have to specify output `resolution`.